presubmit-service/package.json

```diff
@@ -1,13 +1,11 @@
 {
-  "name": "presubmit-service",
-  "version": "0.0.1",
   "private": true,
-  "engines": {
-    "node": ">=8.0.0"
-  },
   "scripts": {
-    "start": "node ./index.js",
-    "gcp-build": "tsc index.ts"
+    "start": "node ./dist/index.js",
+    "build": "rm -rf dist/ && tsc -b tsconfig.json",
```

Comment:
What does `-b` do? I usually just do `tsc -p`

Reply:
It builds the project and its dependencies if they are out of date, but `tsc -p tsconfig.json` is probably more appropriate.